ga/23.0.0.6/full/Dockerfile.ubuntu.ibmjava8

```diff
@@ -12,38 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM websphere-liberty:23.0.0.6-kernel-java8-ibmjava AS installBundle
-ARG VERBOSE=false
-ARG REPOSITORIES_PROPERTIES=""
+FROM websphere-liberty:kernel-java8-ibmjava
 
-# If there is a local copy of the repository use that instead
-COPY resources/ /tmp/
-
-# Install the base bundle
-RUN set -eux; \
-  if [ ! -z "$REPOSITORIES_PROPERTIES" ]; then \
-    mkdir /opt/ibm/wlp/etc/; \
-    echo "$REPOSITORIES_PROPERTIES" > /opt/ibm/wlp/etc/repositories.properties; \
-    installUtility install --acceptLicense baseBundle; \
-    rm /opt/ibm/wlp/etc/repositories.properties; \
-  elif [ -f /tmp/wlpRepo.zip ]; then \
-    installUtility install --acceptLicense baseBundle --from=/tmp/wlpRepo.zip; \
-  else \
-    installUtility install --acceptLicense baseBundle; \
-  fi; \
-  rm -rf /output/workarea /output/logs; \
-  find /opt/ibm/wlp ! -perm -g=rw -print0 | xargs -r -0 chmod g+rw;
+USER root
 
-FROM websphere-liberty:23.0.0.6-kernel-java8-ibmjava
 ARG VERBOSE=false
+ARG REPOSITORIES_PROPERTIES=""
 
-# Copy the runtime
-COPY --from=installBundle /opt/ibm/wlp /opt/ibm/wlp
+# Install the base bundle
```

Comment:
This is reverting back to the broken code that prevents REPOSITORIES_PROPERTIES from working.

Reply:
should be resolved now